modules/junit-jupiter/src/main/java/org/testcontainers/junit/jupiter/TestcontainersExtension.java

```diff
@@ -102,25 +164,15 @@ boolean isDockerAvailable() {
     }
 
     private Stream<StoreAdapter> findSharedContainers(Class<?> testClass) {
-        return ReflectionUtils.findFields(
-                testClass,
-                isSharedContainer(),
-                ReflectionUtils.HierarchyTraversalMode.TOP_DOWN)
-            .stream()
-            .map(f -> getContainerInstance(null, f));
+        return ReflectionUtils.findFields(testClass, isSharedContainer(), ReflectionUtils.HierarchyTraversalMode.TOP_DOWN).stream().map(f -> getContainerInstance(null, f));
     }
 
     private Predicate<Field> isSharedContainer() {
         return isContainer().and(ReflectionUtils::isStatic);
     }
 
     private Stream<StoreAdapter> findRestartContainers(Object testInstance) {
-        return ReflectionUtils.findFields(
-                testInstance.getClass(),
-                isRestartContainer(),
-                ReflectionUtils.HierarchyTraversalMode.TOP_DOWN)
-            .stream()
-            .map(f -> getContainerInstance(testInstance, f));
+        return ReflectionUtils.findFields(testInstance.getClass(), isRestartContainer(), ReflectionUtils.HierarchyTraversalMode.TOP_DOWN).stream().map(f -> getContainerInstance(testInstance, f));
```

Comment:
Please can the original formatting be restored?

Reply:
Absolutely. I don't know how this could happen.